build.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+if [ ! -d "build" ]; then
+	svn co https://plugins.svn.wordpress.org/php-compatibility-checker build
+else
+	pushd build
+	svn up
+	popd
+fi
```

Comment:
I felt that it was important to build the zip with the actual code found in the SVN repo.